Wabbajack/View Models/Gallery/ModListGalleryVM.cs

```diff
@@ -102,7 +123,13 @@ public ModListGalleryVM(MainWindowVM mainWindowVM)
 
             OnlyInstalledCheckedCommand = ReactiveCommand.Create(() =>
             {
-                GameType = ALL_GAME_TYPE;
+                if (OnlyInstalled)
+                {
+                    GameType = ALL_GAME_TYPE;
```

Comment:
I think GameType could also be "reset" via Rx instead of making an `OnlyInstalledCheckedCommand` member.  Just do an Rx subscription to OnlyInstalled as shown above, and reset GameType to All at that point.